pkg/plugin/kubernetes.go

```diff
@@ -149,20 +154,12 @@ func getDeployment(clientset kubernetes.Interface, namespace string, opts *local
 
 // ensureDeploymentHasVolume check the velero deployment for a matching Volume name
 // and if it does not exist, adds it to the podspec.
-func ensureDeploymentHasVolume(deployment *appsv1.Deployment, volumeSpec *corev1.Volume, volumeMountSpec *corev1.VolumeMount, opts *localVolumeObjectStoreOpts) error {
+func ensureDeploymentHasVolume(deployment *appsv1.Deployment, volumeSpec *corev1.Volume, volumeMountSpec *corev1.VolumeMount) error {
 
 	// If the volume name is the same, but the path is different, we should fix the path in place
 	if exists, idx := podHasDuplicateVolumeName(&deployment.Spec.Template.Spec, volumeSpec); exists {
 		deployment.Spec.Template.Spec.Volumes[idx] = *volumeSpec
 	} else {
-		podSecurityCxt, err := getPodSecurityContext(opts)
-		if err != nil {
-			return errors.Wrap(err, "unable to get security context")
-		}
-		if podSecurityCxt != nil {
-			deployment.Spec.Template.Spec.SecurityContext = podSecurityCxt
-		}
```

Comment:
This security context was already being set as part of `ensureDeploymentHasConfigAndFileserver`